docs/advanced_topics/images/image_formats.rst

```diff
@@ -0,0 +1,49 @@
+Image formats
+=========================
+
+Using the picture element
+-------------------------
+
+The `picture element <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture>`_
+can be used with wagtails ``format-<type>`` image operation to specify different
+image formats and let the browser choose it's preferred one.
+
+
+To do this with wagtail:
```

Comment:
Wagtail

Reply:
I feel like this line is already implied by the fact that it's in the Wagtail docs, could we use "For example:" instead?